### PR TITLE
Speedup setup build env

### PR DESCRIPTION
**Pull request by arturrez**
resolves: https://github.com/ava-labs/avalanche-cli/issues/1596 
speedup: 103s -> 95s for 2 nodes cluster

`./bin/avalanche node create lt0 --use-static-ip=false --latest-avalanchego-version=true --aws  --region us-east-1 --num-validators=2 --devnet --num-apis=0  --node-type=c5.2xlarge --skip-monitoring`

### CODE REVIEW

File: pkg/ssh/ssh.go [hunk 1]
```diff
@@ -429,13 +430,36 @@ func RunSSHUpdateSubnet(host *models.Host, subnetName, importPath string) error
...
 		constants.SSHScriptTimeout,
 		"shell/setupBuildEnv.sh",
-		scriptInputs{GoVersion: constants.BuildEnvGolangVersion},
-	)
+		scriptInputs{},
+	); err != nil {
+		return err
+	}
+	eg := &errgroup.Group{}
+	eg.SetLimit(2)
```
Comment (patrick-ogrady) [inline pkg/ssh/ssh.go:443]: I don't think you need to SetLimit for this?

File: pkg/ssh/shell/setupBuildEnv.sh [hunk 2]
```diff
@@ -3,22 +3,3 @@ set -e
 #name:TASK [install gcc if not available]
 sudo apt-get -y -o DPkg::Lock::Timeout=120 update
```
Comment (patrick-ogrady) [inline pkg/ssh/shell/setupBuildEnv.sh:4]: I wonder if you can do this concurrently? I don't think it'll be used by the installs?

Comment (arturrez) [reply pkg/ssh/shell/setupBuildEnv.sh:4]: we can delete this actually as we do it in pkg/ssh/shell/setupNode.sh

File: pkg/ssh/ssh.go (hunk 1 above)
Comment (arturrez) [reply pkg/ssh/ssh.go:443]: sure.

File: pkg/ssh/shell/setupBuildEnv.sh (hunk 2 above)
Comment (arturrez) [reply pkg/ssh/shell/setupBuildEnv.sh:4]: no we can't unfortunately

**Comment by arturrez** [comment]
closing in favour of https://github.com/ava-labs/avalanche-cli/pull/1608

**Pull request closed without merging**